Cargo.toml

```diff
@@ -11,14 +11,15 @@ keywords = ["bevy", "wasm", "web", "gamedev"]
 categories = ["game-development", "game-engines"]
 license = "MIT OR Apache-2.0"
 
+
 [package.metadata.docs.rs]
 all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
 default-target = "wasm32-unknown-unknown"
 
 [features]
-default = ["worker"]
+default = []
```

Comment:
i think default can be removed when empty?

Reply:
Correct, it can.